### PR TITLE
Add new cmdlet to query port mapping and update existing inboundNatRule cmdlets

### PR DESCRIPTION
**Pull request by Feifei-Z**
## Description

Added new cmdlet to support query Load Balancer inbound nat rule port mapping lists for backend addresses. Also updated cmdlets to support inbound nat rule V2 configurations

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1074
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/LoadBalancer/GetAzureLoadBalancerBackendAddressInboundNatRulePortMapping.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "LoadBalancerBackendAddressInboundNatRulePortMapping"), OutputType(typeof(PSInboundNatRulePortMapping))]
```
Comment (wyunchi-ms) [inline src/Network/Network/LoadBalancer/GetAzureLoadBalancerBackendAddressInboundNatRulePortMapping.cs:30]: Please define a default parameter set here.

File: src/Network/Network.Test/ScenarioTests/LoadBalancerTests.ps1 [hunk 2]
```diff
@@ -1096,6 +1096,172 @@ function Test-LoadBalancerChildResource
     }
 }
 
+<#
```
Comment (wyunchi-ms) [inline src/Network/Network.Test/ScenarioTests/LoadBalancerTests.ps1:1099]: May need to update the test code for `Test-LoadBalancerChildResource`.

Comment (Feifei-Z) [reply src/Network/Network.Test/ScenarioTests/LoadBalancerTests.ps1:1099]: Already test inboundNatRule v2 in Test-LoadBalancerInboundNatRuleV2-InternalLB and Test-LoadBalancerInboundNatRuleV2. If update Test-LoadBalancerChildResource, that will just add the same code

File: src/Network/Network/LoadBalancer/GetAzureLoadBalancerBackendAddressInboundNatRulePortMapping.cs (hunk 1 above)
Comment (Feifei-Z) [reply src/Network/Network/LoadBalancer/GetAzureLoadBalancerBackendAddressInboundNatRulePortMapping.cs]: added

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**